tg_bot.py

```diff
@@ -108,9 +119,113 @@ def quering_termins(update, context, reuse=False):
     if not found_any:
         msg.reply_text('Unfortunately, everything is booked. Please come back in several days :(')
 
+    buttons = [InlineKeyboardButton(text="Schedule", callback_data="schedule"),
+               InlineKeyboardButton(text="No, return back", callback_data="return")]
+    custom_keyboard = [buttons]
+
+    msg.reply_text(
+        'If you want, you can schedule checking appointments of this type by interval',
+        reply_markup=InlineKeyboardMarkup(custom_keyboard, one_time_keyboard=True))
+
+    return SCHEDULE_APPOINTMENT
+
+
+def set_retry_interval(update, context):
+    if update.callback_query and update.callback_query.data == 'return':
+        return selecting_buro(update, context)
+    else:
+        msg = update.callback_query.message if update.callback_query else update.message
+        msg.reply_text('Please type interval in minutes. Interval should greater or equals 15 minutes.')
+        return SELECT_INTERVAL
+
+
+def print_available_termins(update, context):
+    """
+    Checks for available termins and prints them if any
+    """
+    department = context.user_data['buro']
+    termin_type_str = context.user_data['termin_type']
+
+    msg = update.message
+
+    appointments = termin.get_termins(department, termin_type_str)
+    found_any = False
+    for k, v in appointments.items():
+        for date in v['appoints']:
+            if v['appoints'][date]:
+                found_any = True
+                msg.reply_text('Available appointments on date %s are %s' % (date, '\n'.join(v['appoints'][date])))
```

Comment:
Please add printing `caption` since it's possible to have termins from several "blocks" of the buro
![image](https://user-images.githubusercontent.com/918446/62315378-7d7d4800-b495-11e9-9dbb-5c35250c035c.png)

Reply:
done